Sources.cmake

```diff
@@ -358,13 +358,19 @@ target_sources(isocline PRIVATE
     extern/isocline/src/isocline.c
 )
 
-# Common sources shared between all CLI apps
-target_sources(Luau.CLI.lib PRIVATE
-    CLI/FileUtils.cpp
-    CLI/Flags.cpp
-    CLI/Flags.h
-    CLI/FileUtils.h
-)
+
+if (TARGET Luau.Repl.CLI OR TARGET Luau.Analyze.CLI OR
+    TARGET Luau.Ast.CLI OR TARGET Luau.CLI.Test OR
+    TARGET Luau.Reduce.CLI OR TARGET Luau.Compile.CLI OR
+    TARGET Luau.Bytecode.CLI)
+    # Common sources shared between all CLI apps.
+    target_sources(Luau.CLI.lib PRIVATE
+        CLI/FileUtils.cpp
+        CLI/Flags.cpp
+        CLI/Flags.h
+        CLI/FileUtils.h
+    )
+endif()
```

Comment:
Why did you revert this?

Reply:
I actually don't know why it auto merged like that - I did not get a merge conflict or anything. Manually added this back.